util/collections_test.go

```diff
@@ -5,6 +5,28 @@ import (
 	"testing"
 )
 
+func TestMatchesAny(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		list     []string
+		element  string
+		expected bool
+	}{
+		{nil, "", false},
+		{[]string{}, "", false},
+		{[]string{}, "foo", false},
+		{[]string{"foo"}, "kafoot", true},
+		{[]string{"bar", "foo", ".*Failed to load backend.*TLS handshake timeout.*"}, "Failed to load backend: Error...:...  TLS handshake timeout", true},
+		{[]string{"bar", "foo", ".*Failed to load backend.*TLS handshake timeout.*"}, "Failed to load backend: Error...:...  TLxS handshake timeout", false},
```

Comment:
It may be worth including some real-world error messages to verify that the RegEx doesn't get tripped up on any special characters.

Reply:
This was a good suggestion. Tested with real world error messages and learned that I had to add the (?s) flag in the regexp to handle multi-line.